Tests/EventListener/FinishRootSpanSubscriberTest.php

```diff
@@ -33,25 +39,23 @@ public function testGetSubscribedEvents(): void
         self::assertArrayHasKey('kernel.finish_request', $this->subject::getSubscribedEvents());
     }
 
-    public function testOnFinishRequestIsMasterRequest(): void
+    public function testOnFinishRequestIsMainRequest(): void
     {
-        $event = $this->prophesize(KernelEvent::class);
-        $event->isMasterRequest()->willReturn(true);
+        $event = new KernelEvent($this->kernel->reveal(), $this->request->reveal(), HttpKernelInterface::MASTER_REQUEST);
```

Comment:
Maybe add a comment that once Symfony 4.4 is no longer supported, we should switch to HttpKernelInterface::MAIN_REQUEST

Reply:
Good point, added the comment.